components/rhel/README_CDK_ZIP.rst

```diff
@@ -15,6 +15,8 @@ Here is the mapping of upstream projects and READMEs for CDK.zip file shipped wi
  ├── plugins
  │   ├── vagrant-registration.gem
  │   ├── vagrant-registration-README.md
+ │   ├── vagrant-sshfs-1.1.0.gem
```

Comment:
Let's remove the version info like you are doing for `vagrant-registration` because this file only to map what available in the zip otherwise we have to create PR everytime when there is version update.